cypress/integration/changeAccount.spec.js

```diff
@@ -0,0 +1,141 @@
+/// <reference types="Cypress" />
+
+import { login } from "../page_object/login.js";
+import { account } from "../page_object/account.js";
+import validation from "../../validationMessages.json";
+import data from "../fixtures/data.json";
+import updatedPassword from "../fixtures/updatedPassword.json"
+
+
+let faker = require("faker");
+
+let accountData = {
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName()
+}
+
+let passwords = {
+    invalidCurrentPassword: "random123",
+    invalidShortPassword: "1",
+    validPassword: "pass1234"
+}
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+  })
+
+describe("Changing account settings", () => {
+    beforeEach(() => {
+        cy.visit("/login");
+        if(updatedPassword.updatedPassword){
+            return login.loginClick(data.user.email, updatedPassword.updatedPassword);
+        }        
+        cy.url().should("contain", "my-organizations");
+    })
+    it("Change personal info with valid values", () => {
+        cy.intercept("PUT", "api/v2/users").as("updateProfileInfo");
+        account.changeAccountSettings();
+        account.changePersonalInfo(accountData.firstName, accountData.lastName);
+        cy.wait("@updateProfileInfo").then((intercept) => {
+            console.log(intercept);
+            expect(intercept.response.body.first_name).to.equal(accountData.firstName);
```

Comment:
Ne bih previse paznje pridavao asertacijama unutar `cy.intercept()` jer je to izvan domena E2E, iako je korisno imati.
Ukoliko mozes da potvrdis neki data preko front dela aplikacije, kao sto u ovom slucaju mozes npr. da refreshujes stranu i napises asertaciju da je update-ovana vrednost unutar odgovarajuceg inputa (tu bi bila jos jedna lepa primena `cy.invoke()`).
Sa asertacijama nad `intercept` objektom bih se zadrzao samo na status kodu, u svrhu cekanja da se request izvrsi i da ne padaju testovi zbog API poziva.

Reply:
:(((